examples-api-use/flask/static/js/canvas-func.js

```diff
@@ -0,0 +1,99 @@
+
+
+function init_canvas(w, h)
+{
+    
+
+
+    var errMsgDiv = document.getElementById('errMsg');
+    var canvas = document.getElementById('ftCanvas');
+    var ctx = canvas.getContext("2d");
+    var webgl = canvas.getContext("experimental-webgl");
+    function animateFunction(func) {
+        fn = function () {
+            var d = new Date();
+            var t = d.getTime();
+            func(ctx, webgl, t);
+
+            requestAnimationFrame(fn);
+

```

Comment:
.. or here a free line at the end of a block.

There are more examples below, e.g in animFunc() are two newlines